core/src/main/java/org/bitcoinj/wallet/DefaultCoinSelector.java

```diff
@@ -87,12 +95,12 @@ public CoinSelection select(Coin target, List<TransactionOutput> candidates) {
     /** Sub-classes can override this to just customize whether transactions are usable, but keep age sorting. */
     protected boolean shouldSelect(Transaction tx) {
         if (tx != null) {
-            return isSelectable(tx);
+            return isSelectable(tx, network);
         }
         return true;
     }
 
-    public static boolean isSelectable(Transaction tx) {
+    public static boolean isSelectable(Transaction tx, Network network) {
```

Comment:
This is public API, on purpose. I know it will break because of the new parameter, but reducing its visibility exclude it from being used at all.

See https://github.com/bitcoin-wallet/bitcoin-wallet/blob/721acfdfe3457a67c11681f5775fbef7d0e69a71/wallet/src/de/schildbach/wallet/ui/TransactionsAdapter.java#L153

Reply:
I think this is a candidate for further refactoring in the future.  (For example it's weird that a static method requires the context and the transaction confidence table, but again this is something we will deal with later.)